staging/src/k8s.io/code-generator/_examples/apiserver/apis/example2/types.go

```diff
@@ -22,6 +22,7 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 
 // TestType is a top-level type. A client is created for it.
+// +genclient:nonNamespaced
 type TestType struct {
```

Comment:
nit: maybe remove the blank line #23, and switch the added line 25 with 24?

Reply:
Done.